### PR TITLE
Add system-out, properties to JUnit output

### PR DESCRIPTION
**Pull request by spenced**

### CODE REVIEW

File: src/testdrive/junit.py [hunk 1]
```diff
@@ -117,7 +151,13 @@ def junit(suite, cases, hostname=None, prettify=False):
         time - test duration in seconds
 
     If `timestamp` is supplied then `time` must also be supplied.
+
+    If both `baseurl_ids` and `baseurl_specs` are supplied then add a property
+    for 'test_specification' by mapping a URL under `baseurl_specs`.
     """
+    uri_builder = None
+    if baseurl_ids and baseurl_specs:
```
Comment (crwr45) [inline src/testdrive/junit.py:167]: If either, or both, are set, then check they are valid URLs

File: src/testdrive/junit.py [hunk 2]
```diff
@@ -117,7 +151,13 @@ def junit(suite, cases, hostname=None, prettify=False):
         time - test duration in seconds
 
     If `timestamp` is supplied then `time` must also be supplied.
+
+    If both `baseurl_ids` and `baseurl_specs` are supplied then add a property
+    for 'test_specification' by mapping a URL under `baseurl_specs`.
```
Comment (crwr45) [inline src/testdrive/junit.py]: rephrase this to make it clearer that it's essentially a find+replace to get the spec url.

Comment (spenced) [reply src/testdrive/junit.py]: Rephrased

**Review by crwr45 — APPROVED** [review]
